crates/trippy-core/src/strategy.rs

```diff
@@ -1191,7 +1191,7 @@ mod state {
             self.buffer[usize::from(resp.sequence - self.round_sequence)] =
                 ProbeStatus::Complete(completed);
 
-            // If this `ProbeStatus` found the target then we set the `target_tll` if not already
+            // If this `ProbeStatus` found the target then we set the `target_ttl` if not already
```

Comment:
I changed this because below it's written as `ttl`/`target ttl` --
There is a networking concept of `target link layer`, but there's no indication that this is the intended item.